.globality/build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "python-library",
+  "params": {
+    "name": "microcosm-elasticsearch",
+    "app_name": "example",
+    "database": "elasticsearch"
```

Comment:
Should we have the `ES` version here?

Reply:
If a specific version is needed, a param of `"elasticsearch_version"` needs to be given with a version that corresponds to the right Docker container tag. The current default is `5.3.1`.